### PR TITLE
Tf 13 required blocks migration readme update

### PR DESCRIPTION
**Pull request by Smuggla**

### CODE REVIEW

**Comment by Smuggla** [comment]
Addresses https://github.com/carlpett/terraform-provider-sops/issues/44

File: README.md [hunk 1]
```diff
@@ -147,3 +147,32 @@ Click the `Allow Anyway` button.
 Building and testing is most easily performed with `make build` and `make test` respectively.
 
 The PGP key used for encrypting the test cases is found in `test/testing-key.pgp`. You can import it with `gpg --import test/testing-key.pgp`.
+
+## Transitioning to Terraform 0.13 (0.12.20+) provider required blocks.
+
+With Terraform 0.13 (0.12.20+), providers are available/downloaded via the [terraform registry](https://registry.terraform.io/providers/carlpett/sops/latest) via a required_providers block (Yay!)
```
Comment (carlpett) [inline README.md:153]: I think we can just call this 0.13 for simplicity. Even though 0.12.20+ formally supports the `source` field, it appears to panic if you set `source`: https://github.com/hashicorp/terraform/blob/v0.12.20/configs/module.go#L173-L178
(And the same behaviour is kept up until 0.12.29 at least)

File: README.md [hunk 2]
```diff
@@ -147,3 +147,32 @@ Click the `Allow Anyway` button.
...
+  required_version = ">= 0.13.0"
+  required_providers {
+    sops = {
+      source = "carlpett/sops"
+      version = "0.5.1"
+    }
+  }
+}
+```
+A prerequisite when converting is that you must from remove the data source block from the previous SOP provider in your `terraform.state` file.
```
Comment (carlpett) [inline README.md:166]: Nit: Typo in SOPS :)

Comment (carlpett) [reply README.md:166]: Is there a high-level description we can add here on the method? I suppose you did some careful `terraform state rm ...`?

**Comment by Djiit** [comment]
This worked for me. We should merge it. @Smuggla do you think you'll have time to do the small requested changes ? Many thanks!

File: README.md (hunk 2 above)
Comment (toninis) [reply README.md:166]: Terraform (at least v0.13.4) prompts you do run this command when you a change the provider
```bash
terraform state replace-provider registry.terraform.io/-/sops registry.terraform.io/carlpett/sops
```
FYI

Comment (Nastaliss) [reply README.md:166]:
```suggestion
A prerequisite when converting is that you must remove the data source block from the previous SOP provider in your `terraform.state` file. 
```
Extra 'from' in here :)

**Comment by Smuggla** [comment]
Apologies was between computers... 

I have updated as requested.

Thanks!

**Comment by carlpett** [comment]
Sorry, I had missed the update here. Thanks for the help! :pray:

**Pull request merged**